connector_dns/models/dns.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Elico Corp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import models, fields, api
+
+
+class DNSBackend(models.Model):
+    _name = 'dns.backend'
+    _inherit = 'connector.backend'
+    _backend_type = 'dns'
+
+    def _select_version(self):
+        return []
+
+    login = fields.Char(
+        string='Login',
+        help="Provider's login.",
+        required=True
+    )
+    password = fields.Char(
+        string='Password',
+        help="Provider's password.",
+        required=True
+    )
+    state = fields.Selection(
+        [('draft', 'Draft'), ('done', 'Done'),
+         ('exception', 'Exception')],
+        string='State',
+        default="draft",
+        help='"Confirmed" when the domain has been succesfully created.'
+    )
+    version = fields.Selection(
+        selection='_select_version',
+        string='Service Provider',
+        help='DNS service provider',
+        required=True
+    )
+
+    @api.multi
+    def name_get(self):
+        res = []
+        for backend in self:
+            res.append((backend.id, '%s (%s)' % (backend.name, backend.login)))
+        return res
+
+
+class DNSDomain(models.Model):
```

Comment:
Hmmmm if you're using this in production maybe we leave a change like that out in order to avoid migrations. I changed the `openerp_id` to `odoo_id` too - with the specific intent to ease future migrations, but I didn't realize you guys are live already. I can reverse the breaking fixes.